files/generate_ansible_command.py

```diff
@@ -289,6 +302,9 @@ def get_hostname(x):
     else:
         commands_to_run.append('ansible-playbook -D %s' % p)
 
+for l in local_playbooks_to_run:
+    commands_to_run.append('sudo /usr/local/bin/ansible_local.py %s' % l)
```

Comment:
As said in the doc, this reflect the way it work on ansible-pull, so this change if the upstream tool change, and I think it didn't change in 3 years.

The ansible_local check is to make sure people with sudo do insert a random playbook or do path traversal. The check in this part of the code is to detect if anything should be run. We could create a whole library just to refactor that code, but that's kinda overkill.  I would take it if upstream (ansible) did export that as a stable function however.

I also pondered on putting the argument verification in sudo, but this might not work well if the name of the server change (even if I couldn't find a problem, it just make cumbersome to do).